### PR TITLE
Gem packaging workshop

### PR DESCRIPTION
**Pull request by craigjbass**

### CODE REVIEW

File: workshops/04-Gem-Packaging-Workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Gem Packaging
+
+In Ruby we use Gems to distribute reusable components.
+
+For example, `rspec` is distributed as a Gem.
```
Comment (zuren) [inline workshops/04-Gem-Packaging-Workshop/README.md]: More examples of gems might be useful, as I feel the phrase "reusable components" may be difficult to understand? What's a component, why would I reuse it?

Comment (danielburnley) [reply workshops/04-Gem-Packaging-Workshop/README.md]: Maybe linking to their git repos too? Though i'd pick some smol simple gems for that

File: workshops/04-Gem-Packaging-Workshop/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+When we install a gem by invoking `gem install rspec` we download a package from RubyGems.org.
```
Comment (kjdchapman) [inline workshops/04-Gem-Packaging-Workshop/README.md]: This language might imply that you have to go to rubygems.org and sort it out yourself.

Maybe:
> When we install a gem by invoking `gem install rspec`, it'll download and install a package (from RubyGems.org) for you.

File: workshops/04-Gem-Packaging-Workshop/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+If we are using Bundler with a Gemfile, we are able to download a specific version of a Gem. This version management is important for the stability of your project.
+
+## Creating your first library
+
+We're going to be creating a Gem to allow anyone to wrap text onto new lines.
+
+We need a name for our Gem - and we need to make sure it's not already in use so check [rubygems.org](https://rubygems.org/) to make sure.
```
Comment (kjdchapman) [inline workshops/04-Gem-Packaging-Workshop/README.md]: make sure twice reads weirdly, can probably miss the second'un.

**Pull request merged**